### ISSUE TITLE
Input box cannot be focused using TalkBack screen reader in web version of vscode in Android operating system

### ISSUE THREAD

**Issue by rahulbhai9** [microsoft/vscode#198173]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 
- OS Version: 

If someone tries to use vscode web version in the Android phone with screen reader they are unable to navigate to the input box for writing a code. Somehow it is not focusable with screen reader. Steps to Reproduce:

1. Open web version of vscode in any Android phone
2. 
Use screen reader like TalkBack and try to navigate to the input box where code can be written.
